run.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+poetry install
```

Comment:
You already ran poetry install in the Dockerfile.

If you found that Heroku was failing to install them in a virtual environment correctly, you should update your Dockerfile to do `RUN poetry config virtualenvs.create false --local && poetry install`

Also, for production, you should do poetry install --no-dev to skip things like pytest if they were added correctly.